module/Kanbanize/src/Kanbanize/Controller/BoardsController.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace Kanbanize\Controller;
+
+use Application\Controller\OrganizationAwareController;
+use Application\View\ErrorJsonModel;
+use People\Service\OrganizationService;
+use People\Organization;
+use Kanbanize\Service\KanbanizeAPI;
+use Kanbanize\Service\KanbanizeApiException;
+use Kanbanize\Service\Importer;
+use TaskManagement\Service\StreamService;
+use TaskManagement\Task;
+use Zend\Filter\FilterChain;
+use Zend\Filter\StringTrim;
+use Zend\Filter\StripNewlines;
+use Zend\Filter\StripTags;
+use Zend\View\Model\JsonModel;
+use Zend\Validator\InArray;
+use Zend\Validator\NotEmpty;
+use Kanbanize\KanbanizeStream;
+use Kanbanize\Service\KanbanizeService;
+
+
+class BoardsController extends OrganizationAwareController{
+
+	protected static $resourceOptions = ['POST', 'GET'];
+	protected static $collectionOptions= [];
+	/**
+	 * @var StreamService
+	 */
+	private $streamService;
+	/**
+	 * @var KanbanizeAPI
+	 */
+	private $client;
+	/**
+	 * @var KanbanizeService
+	 */
+	private $kanbanizeService;
+
+	public function __construct(OrganizationService $organizationService, StreamService $streamService, KanbanizeAPI $client, KanbanizeService $kanbanizeService){
+		parent::__construct($organizationService);
+		$this->streamService = $streamService;
+		$this->client = $client;
+		$this->kanbanizeService = $kanbanizeService;
+	}
+
+	public function invoke($id, $data){
+		if(is_null($this->identity())) {
+			$this->response->setStatusCode(401);
+			return $this->response;
+		}
+		if(!$this->isAllowed($this->identity(), $this->organization, 'Kanbanize.BoardSettings.create')) {
+			$this->response->setStatusCode(403);
+			return $this->response;
+		}
+		$notEmptyValidator = new NotEmpty();
+		$error = new ErrorJsonModel();
+		if(!(isset($data['projectId']) && $notEmptyValidator->isValid($data['projectId']))){
```

Comment:
Hai una negazione (`!`) di un and (`&&`), quindi si deve trasformare nell'or (`||`) di due negazioni (`!isset`) (`!$n...->isValid()`)